import_logs.py

```diff
@@ -459,6 +459,11 @@ def get(self, key):
 _NCSA_EXTENDED_LOG_FORMAT = (_COMMON_LOG_FORMAT +
     r'\s+"(?P<referrer>.*?)"\s+"(?P<user_agent>.*?)"'
 )
+
+
+_GANDI_SIMPLE_HOSTING_FORMAT = (
+    r'(?P<host>[0-9a-zA-Z-_.]*)\s(?P<ip>[a-zA-Z0-9.]*)\s(\S*)\s(?P<userid>\S*)\s\[(?P<date>.*?)\s+(?P<timezone>.*?)\]\s\((?P<generation_time_secs>[0-9a-zA-Z\s]*)\)\s"(?P<method>[A-Z]*)\s(?P<path>[^\s]*)\s([^\s]*)"\s(?P<status>[0-9]*)\s(?P<length>[^\s]*)\s"(?P<referrer>[^\s]*)"\s"(?P<user_agent>[^"]*)"'
+)
```

Comment:
Hi @tups, is there a reason some groups here aren't named groups? And is it possible for the whitespace parts to use `\s+` like the other regexes, as well as the fields we don't care about be parsed with `\S+`?

Reply:
Hi, I just checked, it is actually not necessary to capture this. It corresponds to the format %l of the apache log: https://httpd.apache.org/docs/2.4/fr/logs.html
I just improved the RegEx with your recommendations